gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/map/MergeEdge.feature

```diff
@@ -151,7 +151,7 @@ Feature: Step - mergeE()
       g.addV("person").property("name", "marko").
         addV("person").property("name", "vadas")
       """
-    And using the parameter xx1 defined as "m[{\"t[label]\": \"knows\", \"D[OUT]\":\"v[marko]\", \"D[IN]\":\"v[vadas]\"}]"
+    And using the parameter xx1 defined as "m[{\"t[label]\": \"knows\", \"D[OUT]\":\"v[marko].id\", \"D[IN]\":\"v[vadas].id\"}]"
```

Comment:
These mergeE cases worry me. Once this gets merged up to TinkerPop 4 and GraphTraversal starts generating GremlinLang instead of Bytecode, Vertex will no longer be allowed in the mergeE merge map. This would be an unintended breaking change.
It's a bit messy but I wonder if we need to add that same sort of `Vertex -> Id` syntax sugar in the mergeE map at the GraphTraversal level in each GLV similar to what has been done in from() and to().
Thoughts?

Reply:
After further consideration, I think it's best that this vertex -> id sugar be excluded from 3.8 dev as it's not necessary for bytecode. I think a seperate PR which includes such sugar for master might be warranted here.